core/commonMain/src/kotlinx/serialization/builtins/BuiltinSerializers.kt

```diff
@@ -251,6 +252,20 @@ public fun UShort.Companion.serializer(): KSerializer<UShort> = UShortSerializer
  */
 public fun Duration.Companion.serializer(): KSerializer<Duration> = DurationSerializer
 
+/**
+ * Returns serializer for [Uuid].
+ * Serializer operates with a standard UUID string representation, also known as "hex-and-dash" format —
```

Comment:
will serialization occur in uppercase or lowercase?

Reply:
In lowercase, as defined by `Uuid.toString()`